src/renderer/src/views/Home/Settings/ProjectConfig/InviteDeviceModal/SelectDevice.tsx

```diff
@@ -85,7 +85,7 @@ const m = defineMessages({
   },
   noDeviceSuggestion2: {
     id: 'views.Home.Settings.ProjectConfig.InviteDeviceModal.SelectDevice.noDeviceSuggestion2',
-    defaultMessage: 'Make sure both devices are on the same version of Mapeo',
+    defaultMessage: 'Make sure both devices are using new Mapeo',
```

Comment:
not a grammatical suggestion so feel free to ignore:

```suggestion
    defaultMessage: 'Make sure both devices are using MapeoSuperS',
```